user/keyd/template.py

```diff
@@ -21,4 +21,5 @@
 def post_install(self):
     self.install_license("LICENSE")
     self.install_service(self.files_path / "keyd")
+    self.install_service(self.files_path / "keyd.user")
     self.install_sysusers(self.files_path / "keyd.conf")
```

Comment:
```suggestion
    self.install_service("^/keyd")
    self.install_service("^/keyd.user")
    self.install_sysusers("^/keyd.conf")
```
keyd.conf also probably ought to be renamed to sysusers.conf